### ISSUE TITLE
Websocket not updating

### ISSUE THREAD

**Issue by bltzr** [jamoma/JamomaMax#944]
reviewing #878 I just noticed that the websocket html page doesn't update when the value changes in Max... I thought it did (did it ?)

**Comment by bltzr** [comment]
in other words, I have to reload the page in order to have it reflect the changes of values in Max... also adding @avilleret and @theod as followers

**Comment by avilleret** [comment]
I confirme that, but I thought it was an expected behavior.
Btw I don't know anything about JS and WebSocket so I might not help :-/

**Comment by laugre** [comment]
Yes I confirm that too, normal behavior until today, listen feature was not well implemented in websocket plugin.
I think "Jamomarmot" (remote web controller) should send a listen request on datas when receive the namespace to be notified and updated.
I'm working on it since friday.

**Comment by bltzr** [comment]
Do you mean you’re trying to make it listen ?
Great !!!
Thank you @laugre !

**Comment by laugre** [comment]
I added listen feature to WebSocket plugin and to Jamomarmot.
It works well with the c++ DemoApp, values are automatically updated on web page when changed in DemoApp.
I want to try it in Max but if I launch websocketApp.maxpat (with freshly generated externals and extensions), I get errors in Max (Unable to load object bundle executable). Did you meet it ?
![capture d ecran 2015-10-27 16 25 24](https://cloud.githubusercontent.com/assets/5321589/10762809/d2a0fac0-7cc7-11e5-8326-5be943bfec07.png)

**Comment by bltzr** [comment]
uhuh... I guess we need @theod here !

**Comment by theod** [comment]
what version of Max are you using ?
I would add @tap in case it is relative to a Max symbol
or @jcelerier in case something change in CMake.

**Comment by theod** [comment]
I think it is relative to the last change @tap made about enabling standalone feature.
maybe we should post an error if we detect Max version under 7. ?

**Comment by laugre** [comment]
@theod I'm using Max 7.0.2.
I was using Cmake command from Cmake wiki page.
`cmake -DBUILD_JAMOMAMAX=ON -DJAMOMAMAX_INSTALL_FOLDER="$HOME/Documents/Max 7/Package" ../
make install`
I have the same result if I use the build.sh script 
`./build.sh --clean --multi --install`
Do I have something wrong, should I update to Max latest version ?

**Comment by jcelerier** [comment]
I encounter the same problem with the nightlies starting from October 15 on
Max 7.

**Comment by reno-** [comment]
@laugre , @jcelerier  : You need Max 7.0.6 to use Jamoma from the October 15th

**Comment by laugre** [comment]
Ok thx reno, i try.

**Comment by theod** [comment]
I've just added an error post in case the Max version is earlier than 7.0.6

**Comment by reno-** [comment]
Maybe we could not loading Jamoma library if Max version is earlier than 7.0.6?

**Comment by theod** [comment]
the init function returns after posting the error so I guess the library will not be loaded at all but I'm not sure ...

**Comment by laugre** [comment]
Ok I don't have this loading error anymore with Max 7.0.6. Thx @reno- and @theod !
I tested websocketApp.maxpat and listening is ok. 
(Sliders work better if you call Jamomarmot from a chrome web browser.)

@theod if I check the "listen messages from/to a MIDI controller" (MIDI ?), I get in Max message "WebSocket 0" every time a value is changed. I think it needs to be formated before printed.
But what does this message correspond to ?
